sql/internal/specutil/convert.go

```diff
@@ -222,6 +245,23 @@ func resolveCol(ref *schemaspec.Ref, sch *schema.Schema) (*schema.Column, error)
 	return col, nil
 }
 
+// FromRealm converts a schema.Realm into []sqlspec.Schema and []sqlspec.Table.
+func FromRealm(r *schema.Realm, fn TableSpecFunc) ([]*sqlspec.Schema, []*sqlspec.Table, error) {
+	var (
+		schemas []*sqlspec.Schema
+		tables  []*sqlspec.Table
+	)
+	for _, sch := range r.Schemas {
+		schemaSpec, tableSpecs, err := FromSchema(sch, fn)
+		if err != nil {
+			return nil, nil, fmt.Errorf("specutil: cannot convert schema %q: %w", sch.Name, err)
+		}
+		schemas = append(schemas, schemaSpec)
+		tables = append(tables, tableSpecs...)
+	}
+	return schemas, tables, nil
```

Comment:
why are we breaking it down? why not just pass the realms ? aren't the tables part of the realms?

Reply:
good point. but, currently no "realm" object in sqlspec. 
In any case, this is one of the reasons why specutil is internal, the API is not finalized and still evolving.